tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+    FOR    ${admin_group}    IN    @{admin_groups}
+             ${present}=  Run Keyword And Return Status    Element Should Be Visible   //*[@class="pf-c-select__menu-item pf-m-selected" and contains(text(), "${admin_group}")]
+            IF  ${present} != True
+                Click Button    //*[@class="pf-c-select__menu-item"and contains(text(), "${admin_group}")]
+            END
+    END
+    Press Keys    None    ESC
+
+Add OpenShift Users groups To Data Science Administrators
+    [Documentation]  Add OpenShift Users groups to Data Science administrators
+    [Arguments]     @{user_groups}
+    Set Library Search Order  SeleniumLibrary
+    Click Button    (//button[@class="pf-c-button pf-c-select__toggle-button pf-m-plain"])[${2}]
+
+    FOR    ${user_group}    IN    @{user_groups}
+            ${present}=  Run Keyword And Return Status
+               ...  Element Should Be Visible   //*[@class="pf-c-select__menu-item pf-m-selected" and contains(text(), "${user_group}")]
+             IF  ${present} != True
+                Click Element    //*[@class="pf-c-select__menu-item"and contains(text(), "${user_group}")]
+             END
+    END
+    Press Keys    None    ESC
+
+Check User Management Option Is Available For The User
+    [Arguments]   ${username}  ${password}  ${auth_type}
+    Launch Dashboard  ocp_user_name=${username}  ocp_user_pw=${password}  ocp_user_auth_type=${auth_type}
+    ...               dashboard_url=${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    Menu.Navigate To Page    Settings    User management
+    SeleniumLibrary.Wait Until Element Is Visible   //*[@class="pf-c-button pf-c-select__toggle-button pf-m-plain"]  timeout=20
+
+Remove OpenShift Users Groups To Data Science Administrators
+    [Documentation]     Remove User from OpenShift Users groups to Data Science administrators
+    [Arguments]   @{user_groups}
+    Set Library Search Order  SeleniumLibrary
+    FOR    ${user_group}    IN    @{user_groups}
+            Click Element     (//*[@class="pf-c-form__group-control"])[${2}]//*[@class="pf-c-chip__text" and contains(text(),"${user_groups}")]//following-sibling::button[${1}]
+    END
+    Press Keys    None    ESC
+
+Remove OpenShift Groups To Data Science Administrators
+    [Documentation]  Add OpenShift groups to Data Science administrators
+    [Arguments]     @{admin_groups}
+    Set Library Search Order  SeleniumLibrary
+    FOR    ${admin_group}    IN    @{admin_groups}
+            Click Button    (//*[@class="pf-c-form__group-control"])[${1}]//*[@class="pf-c-chip__text" and contains(text(),"${admin_group}")]//following-sibling::button[${1}]
+    END
+    Press Keys    None    ESC
+
+Save Changes In User Management Setting
+    [Documentation]  Save User management Settings
+    Click Button    Save changes
+    Sleep  60s
+
+Verify Expect Changes In AdminGroups Are Present In CRD
```

Comment:
A more generic keyword name could be: `AdminGroups in OdhDashboardConfig CRD Should Be`